kiwi/xml_state.py

```diff
@@ -1154,6 +1159,10 @@ def get_volumes(self):
                 if ':all' in size:
                     size = None
                     fullsize = True
+                    if have_full_size_volume:
+                        raise KiwiLvmSetupError(
+                            'More than one LVM volume with size="all" attribute'
+                        )
```

Comment:
So this should be covered in a runtime check because we want to fail early. As I was working on the tests anyway I covered this one here:

* https://github.com/SUSE/kiwi/pull/992

You could cherry-pick or delete the if condition here

Reply:
But if the implicit 'size=all' later comes into play, this will not be found by the runtime checker, or does it cover that case also? I guess it only checks the XML file.

Reply:
The implicit size=all should not be set if there is another volume with that configuration.
Actually it would be good if we don't need that, more below